heat/core/tensor.py

```diff
@@ -1360,6 +1362,55 @@ def sqrt(self, out=None):
         """
         return exponential.sqrt(self, out)
 
+    def squeeze(self, axis=None):
+        """
+        Remove single-dimensional entries from the shape of a tensor.
+
+        Parameters:
+        -----------
+        x : ht.tensor
+            Input data.
+
+        axis : None or int or tuple of ints, optional
```

Comment:
more explanation for axis as mentioned before

Reply:
done (dndarray.py)